### ISSUE TITLE
[Feature] removal of AI generated slop, via community reporting

### ISSUE THREAD

**Issue by TuuBiiSan** [ajayyy/SponsorBlock#1963]
as title says, allow the community to report videos as AI generated slop (can also include non-AI content farming slop) so we can just skip em or hide em in our results/recommendations

**Comment by PasttaTypo** [comment]
This was discussed in the discord before. SponsorBlock isn't a tool to hide clickbait-y channels or auto generated slop. It was designed to skip sponsors, plugs, and other promos within videos, and it only extended outside of that to mark videos that are entirely self-promo, sponsor, etc. Basically, SponsorBlock isn't a fact-checker and never will be.
For what it's worth, YouTube actually plans to implement some kind of AI disclosure on videos, ironically powered by AI. [Read More on YT Blog](https://blog.youtube/inside-youtube/our-approach-to-responsible-ai-innovation/)
You can block YT channels from your homepage and video page by going to drop-down and "Don't Recommend Channel." For more extensive blocking, BlockTube and Improve YouTube! are available from the Chrome Store and have options for blocking YT  channels and videos, and Ajay's own DeArrow replaces thumbnails and titles with less sensational ones.
Of course, report channels to YouTube that are violating any policy on spam or misleading practices.

**Comment by Beethoven-n** [comment]
then why does it detect and block filler?

**Comment by Beethoven-n** [comment]
could we get a tag to skip AI-generated sections of videos, though? it seems in-scope, as sponsorblock already has a filler/tangent tag that isn't related to promotions at all

**Comment by E-Kou** [comment]
Yes I would love this.
@PasttaTypo Personally I see SponsorBlock as a tool to perfect the quality of the viewer's watching experience, and marking the AI BS the platform is getting even more full of day-by-day is becoming more and more of a necessity.

**Comment by ghost** [comment]
Yes I'd love that. This low effort slop is really cluttering up the site lately, I wish to not see it. At all. Special in shorts!

**Comment by ajayyy** [comment]
I have pushed the source code of my prototype and am open to help or feedback if anyone wants to help. Not a usable extension, no submission system or anything exists, still just a prototype

https://discord.gg/SponsorBlock

https://github.com/ajayyy/slop-extension
